packages/apps/plugins/plugin-chain/src/ChainPlugin.tsx

```diff
@@ -23,6 +23,7 @@ export const ChainPlugin = (): PluginDefinition<ChainPluginProvides> => {
           [ChainType.typename]: {
             placeholder: ['object placeholder', { ns: CHAIN_PLUGIN }],
             icon: (props: IconProps) => <HeadCircuit {...props} />,
```

Comment:
presumably these are now unused? we should circle back to remove these later I think